apps/protailwind/src/lib/workshops.ts

```diff
@@ -0,0 +1,96 @@
+import groq from 'groq'
+import {sanityClient} from '@skillrecordings/skill-lesson/utils/sanity-client'
+
+const workshopsQuery = groq`*[_type == "module" && moduleType == 'workshop'] | order(_createdAt desc) {
```

Comment:
moved this ove so it would exist, but needs to be reconsidered based on PT context specifically with media loading